sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContentWithSelectionMarks.java

```diff
@@ -0,0 +1,76 @@
+     */
+    public static void main(final String[] args) throws IOException {
+        // Instantiate a client that will be used to call the service.
+        FormRecognizerClient client = new FormRecognizerClientBuilder()
+                                          .credential(new AzureKeyCredential("{key}"))
+                                          .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+                                          .buildClient();
+
+        File sourceFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/java/sample-forms/"
+                                       + "selectionMark/selectionmark_form1.pdf");
+        byte[] fileContent = Files.readAllBytes(sourceFile.toPath());
+        InputStream targetStream = new ByteArrayInputStream(fileContent);
+
+        SyncPoller<FormRecognizerOperationResult, List<FormPage>> recognizeContentPoller =
+            client.beginRecognizeContent(targetStream, sourceFile.length());
+
+        List<FormPage> contentPageResults = recognizeContentPoller.getFinalResult();
+
+        for (int i = 0; i < contentPageResults.size(); i++) {
+            final FormPage formPage = contentPageResults.get(i);
+            System.out.printf("---- Recognized content info for page %d ----%n", i);
+            // Table information
+            System.out.printf("Has width: %.2f and height: %.2f, measured with unit: %s%n", formPage.getWidth(),
+                formPage.getHeight(),
+                formPage.getUnit());
+            final List<FormTable> tables = formPage.getTables();
+            for (int i1 = 0; i1 < tables.size(); i1++) {
+                final FormTable formTable = tables.get(i1);
+                System.out.printf("Table %d has %d rows and %d columns.%n", i1, formTable.getRowCount(),
+                    formTable.getColumnCount());
+                formTable.getCells().forEach(formTableCell -> {
+                    final StringBuilder boundingBoxStr = new StringBuilder();
+                    if (formTableCell.getBoundingBox() != null) {
+                        formTableCell.getBoundingBox().getPoints().forEach(point ->
+                                                                               boundingBoxStr.append(String.format("[%.2f, %.2f]", point.getX(), point.getY())));
+                    }
+                    System.out.printf("Cell has text '%s', within bounding box %s.%n", formTableCell.getText(),
+                        boundingBoxStr);
+                });
+                System.out.println();
+            }
+            // Selection Mark information
+            final List<FormSelectionMark> selectionMarks = formPage.getSelectionMarks();
+            for (int j = 0; j < selectionMarks.size(); j++) {
+                final FormSelectionMark selectionMark = selectionMarks.get(i);
+                System.out.printf("Selection Mark, %s, has state = %s.%n", selectionMark.getText(), selectionMark.getState());
```

Comment:
We should add one example to show the use of `asSelectionMark` too ?

Reply:
selection.getState() -- should this rather be `selectionMark.isSelected()`

Reply:
talked offline. Keep SelectionMarkState, service team will add "undetermined state" in the future.

And added the sample for asSelectionMark in receipt